tables_io/io_layer.py

```diff
@@ -274,6 +319,47 @@ def readHdf5ToTables(filepath):
     return tables
 
 
+def readHdf5ToDataframe(filepath):
+    """
+    Reads `pandas.DataFrame` objects from an hdf5 file.
+
+    Parameters
+    ----------
+    filepath: `str`
+        Path to input file
+
+    Returns
+    -------
+    df : `pandas.DataFrame`
+        The dataframe
+    """
+    if not HAS_HDF5:  #pragma: no cover
+        raise ImportError("h5py is not available, can't read hdf5")
+    if not HAS_PANDAS:  #pragma: no cover
+        raise ImportError("pandas is not available, can't read hdf5 to dataframe")
+    return pd.read_hdf(filepath)
+
+
+def readHdf5Group(infile, groupname=None):
+    """ Get the size of data in a particular group in an hdf5 file
+
+    Parameters
+    ----------
+    infile : `str`
+        File in question
+    groupname : `str` or `None`
+        For hdf5 files, the groupname for the data
+
+    Returns
+    -------
+    FIXME
+    """
+    infp = h5py.File(infile, "r")
+    if groupname != 'None':  #pragma: no cover
```

Comment:
Should be ``groupname is not None`` not ``groupname != 'None'``

Reply:
@sschmidt23 , I grabbed this from RAIL/fileIO.py, was there a reason that you were using 'None' instead of None?

Reply:
Ok, I'll just assume that we can sort this out in RAIL in needs be, and set it to None here.